### PR TITLE
Added API checks and updated README.

### PR DESCRIPTION
**Pull request by joeribekker**
Also did initial work on Docker containers.

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Prerequisites
 
 You need the following libraries and/or programs:
 
-* `Python`_ 3.6 or above
+* `Python`_ 3.7 or above
 * Python `Virtualenv`_ and `Pip`_
 * `PostgreSQL`_ 10 or above
```
Comment (annashamray) [inline INSTALL.rst:23]:
```suggestion
* `PostgreSQL`_ 10 or above with PostGIS extension
```

File: README.rst [hunk 2]
```diff
@@ -27,9 +26,50 @@ To define the format of objects, so called object types, organizations can use
...
+                                `Swagger <https://petstore.swagger.io/?url=https://raw.githubusercontent.com/maykinmedia/objects-api/master/src/openapi.yaml>`_
+==============  ==============  =============================
+
+See: `All versions and changes <https://github.com/maykinmedia/objects-api/blob/master/CHANGELOG.rst>`_
+
+
+Reference implementation
+========================
+
+|build-status| |coverage| |black| |docker| |python-versions|
```
Comment (annashamray) [inline README.rst:47]: 4 of these badges are added above in this file (line 9)

**Review by annashamray — COMMENTED** [review]
I didn't know about github actions before, they look cool!

File: .github/workflows/generate-sdks.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+name: generate-sdks
```
[the remark this answers is not in the document]
Comment (joeribekker) [reply .github/workflows/generate-sdks.yml:1]: Sure!

It's a check that I also added to all reference API implementation at VNG (and Haal Centraal uses it as well) to validate if the openapi.yaml can be used to generate stub code. If this fails, our OAS is either flawed or we make it harder for developers to generate code from it.

Adding this check simply indicates that the OAS can be used to generate code. This is something that wasn't always possible with the SOAP-files in earlier VNG standards (SoapUI would crash for example).

[the remark this answers is not in the document]
Comment (annashamray) [reply .github/workflows/generate-sdks.yml:1]: thank you for explanation

**Review by annashamray — APPROVED** [review]

**Pull request merged**